File: Monstone_Android_Client/app/src/main/java/com/example/nightwingky/monstone/main_program/home/divide/DivideViewHolder.java
package com.example.nightwingky.monstone.main_program.home.divide;

import android.view.View;

import com.example.nightwingky.monstone.R;
import com.example.nightwingky.monstone.main_program.home.TypeAbstractViewHolder;


/**
 * Created by nightwingky on 17-3-18.
 */

public class DivideViewHolder extends TypeAbstractViewHolder<Object> {

    public View view;

    public DivideViewHolder(View itemView) {
        super(itemView);

        this.view = itemView.findViewById(R.id.home_divided_view);
    }

    @Override
    public void bindHolder(Object object) {

    }
}

File: kotlin_client/app/src/main/java/com/graduation/nightwingky/book_drifting/monstone/main_program/fragmentHome/divide/DivideViewHolder.kt
package com.graduation.nightwingky.book_drifting.monstone.main_program.fragmentHome.divide

import android.view.View

import com.graduation.nightwingky.book_drifting.R
import com.graduation.nightwingky.book_drifting.monstone.main_program.fragmentHome.TypeAbstractViewHolder


/**
 * Created by nightwingky on 17-3-18.
 */

class DivideViewHolder(itemView: View) : TypeAbstractViewHolder<Any>(itemView) {

    var view: View

    init {

        this.view = itemView.findViewById(R.id.home_divided_view)
    }

    override fun bindHolder(`object`: Any) {

    }
}
